src/main.ts

```diff
@@ -1,11 +1,17 @@
 import { NestFactory } from '@nestjs/core';
-import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
 import { AppModule } from './app.module';
+import { HttpExceptionFilter } from './http-exception.filter';
 
 const port = process.env.PORT || 3000;
 
 async function bootstrap() {
-    const app = await NestFactory.create(AppModule);
+    const app = await NestFactory.create(AppModule, {
+        logger: ['error', 'warn', 'log', 'debug'],
```

Comment:
die logger level könnte man eigentlich per env setzen. Was meinst du?

```suggestion
        logger: process.env.LOGGER_LEVEL?.split(',') || ['error', 'warn', 'log', 'debug'],
```